### PR TITLE
Add development setup instruction

### PR DESCRIPTION
**Pull request by mysticmode**
cc @gbraad @kpcyrd 

Please review and merge the PR. Thanks.

### CODE REVIEW

**Comment by kpcyrd** [comment]
This might be sufficient (untested, currently on the road):

> - `go get -d github.com/LibreRead/LibreRead/cmd/libreread`
> - `cd $GOPATH/src/github.com/LibreRead/LibreRead`
> - `go run ./cmd/libreread/main.go`

**Comment by mysticmode** [comment]
@kpcyrd I have updated the commands as per your suggestion. It's working.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,17 @@ Self-hosted Free(Libre) Ebook Reader. https://libreread.org
  - Full-text search
  - Highlight & Annotate
  - Supports PDF & EPUB
+ 
+### Production setup
+Please check [this guide](https://github.com/LibreRead/LibreRead/blob/master/docs/INSTALL.md)
```
Comment (gbraad) [inline README.md]: /LibreRead/ ?

File: README.md [hunk 2]
```diff
@@ -28,6 +28,17 @@ Self-hosted Free(Libre) Ebook Reader. https://libreread.org
...
+
+### Development setup
+Install redis and elasticsearch(optional). if you want to use elasticsearch, do `export LIBREREAD_ELASTICSEARCH=1` and run the below commands.
```
Comment (gbraad) [inline README.md]: Install Redis and optionally ElasticSearch

Comment (gbraad) [reply README.md]: run the following commands

Comment (gbraad) [reply README.md]: Which version of go is recommended to be used?

Comment (gbraad) [reply README.md]: How is Redis to be setup? Default settings?

Comment (mysticmode) [reply README.md]: go 1.9

Comment (gbraad) [reply README.md]: @mysticmode You do not need to tell me. The reason I ask, is that this needs to be described... 

This is an onboarding document... the easier it is to get started, the more likely you will get contributions.

Comment (mysticmode) [reply README.md]: Shall I put "Install go 1.9, redis 4 and optionally elasticsearch 5" ?

**Comment by mysticmode** [comment]
@gbraad I have updated the version and mentioned default settings with port info.

Should we need to add contribution guidelines? I'm new to this and I need your help. Thank you.

**Pull request merged**